### DISCUSSION TITLE
Question on dependency version updates

### DISCUSSION THREAD

**Discussion by ramanNarasimhan77** [argoproj/argo-workflows#13281] [Q&A] [answered]
Hello,

I noticed that some version updates done as early as 3-6 months ago do not appear in recent v3.5.8 release

For example,
In `go.mod` - `argo-events` is still 1.7.3 in v3.5.8 but it was upgraded to 1.9.1 in master 3 months ago.
In `ui/package.json` - Updates to package versions of `react-dom`, `react-datepicker` etc done around 6 months ago are missing. React version uplift done last month is also not part of recent release.

I am trying to understand why these version uplifts are missing in latest release.

**Comment by agilgur5** [comment, marked as answer]
> I am trying to understand why these version uplifts are missing in latest release.

We only backport fixes and security patches into patch releases. You'll see the rest in the next minor release.
